weldx/measurement.py

```diff
@@ -207,25 +809,32 @@ def _signal_label(signal):
         )
 
         # draw edge labels
+        edge_labels = {
+            edge: _transformation_label(
+                edge[1], self._graph.edges[edge]["transformation"]
+            )
+            for edge in self._graph.edges
+        }
+
         draw_networkx_edge_labels(graph, positions, edge_labels, ax=axes)
 
         return axes
 
+    def to_tree(self) -> Dict:
+        return dict(
+            name=self._name,
+            data_source=self._source,
+            source_signal=build_tree(self._graph, self._source.name),
+        )
```

Comment:
I'm wondering if we should use the default `nx.DiGraph` serialization here instead of relying on `build_tree`
it would look a little different (more nested) in the schema but maybe more stable and simplier to maintain in the long run

Reply:
I tried this first but then asdf expects a di_graph as root-node. Would need to investigate how the schema has to be adjusted.